dev/tasks/tasks.yml

```diff
@@ -1321,13 +1321,13 @@ tasks:
       image: ubuntu-r-only-r
 
   # This also has -flto=auto
-  test-r-gcc-11:
+  test-r-gcc-12:
     ci: github
     template: docker-tests/github.linux.yml
     params:
       env:
         UBUNTU: 22.04
-        GCC_VERSION: 11
+        GCC_VERSION: 12
```

Comment:
Do we know if we have gcc 11 coverage in a different build? I can look and see if you don't off the top of your head. But if we _don't_ we might want to duplicate these and do both 11 and 12.